atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -2230,19 +2245,21 @@ private void alterGcAndTombstoneInternal(
                                 tableRef.getQualifiedName(),
                                 configManager.getConfig().schemaMutationTimeoutMillis());
                         log.trace("gc_grace_seconds is set to {} for {}.{}",
-                                gcGraceSeconds, keyspace, tableRef);
+                                SafeArg.of("gcGraceSeconds", gcGraceSeconds), UnsafeArg.of("keyspace", keyspace),
+                                UnsafeArg.of("table", tableRef));
                         log.trace("tombstone_threshold_ratio is set to {} for {}.{}",
-                                tombstoneThresholdRatio, keyspace, tableRef);
+                                SafeArg.of("tombstoneThresholdRatio", tombstoneThresholdRatio),
+                                UnsafeArg.of("keyspace", keyspace), SafeArg.of("table", tableRef));
```

Comment:
Assumed this was a mistake and fixed it myself. If I'm wrong, please tell me and I'll fix this in a following pr. :)